mitodl/micromasters
# implement basic learner search UI

#### What are the relevant tickets?

closes #714 

#### What's this PR do?

this implements the basics of the search UI. It puts results on the page, and supports filters on the side.

#### Where should the reviewer start?

Well, there's a lot. I think review all the new components, and it would probably be good to read the code here alongside the [searchkit docs](http://docs.searchkit.co/stable/).

#### How should this be manually tested?

Make sure you have at least one profile that's indexed on your MM install. Then go to `/learners` and look at the results. You should see results, and be able to open and close the filter menus, and add filters.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
